connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/client/util/RemoteSparkSession.scala

```diff
@@ -58,10 +58,15 @@ object SparkConnectServerUtils {
 
   private lazy val sparkConnect: Process = {
     debug("Starting the Spark Connect Server...")
-    val jar = findJar(
+    val connectJar = findJar(
       "connector/connect/server",
       "spark-connect-assembly",
       "spark-connect").getCanonicalPath
+    var driverClassPath = connectJar
```

Comment:
I think we can make `driverClassPath` as `val` like:
```
val driverClassPath = f (IntegrationTestUtils.isCatalystTestJarAvailable) {
    connectJar += ":" ...
} else connectJar
```

Reply:
Thanks for advise. Finished